cmd/virt-handler/virt_launcher.cil

```diff
@@ -1,10 +1,55 @@
+; This is the custom SELinux policy for virt-launcher. This file is hopefully temporary.
+; Applications running in regular container usually have container_t as an SELinux type.
+; However, some applications running in virt-launcher (namely libvirtd) need more permissions.
 (block virt_launcher
-    (blockinherit container)
```

Comment:
It's possible to have multiple blocks in one .cil file. Is there an advantage to merging these blocks? Is there any potential we'll need entries for other components at some point?

Reply:
Yes, it is possible. That would mitigate the second bullet point in my reply to @vladikr (see below). But points 1 and 3 would still be valid.